src/services/character/character.service.ts

```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash'
+import { Query, Document } from 'mongoose'
+import characters from '../../data/models/character-schema'
+import { ICharacter } from '../../types/character/character-type'
+
+export class CharacterService {
+  private static _instance: CharacterService
+
+  public static getInstance(): CharacterService {
+    if(_.isNil(CharacterService._instance)) {
+      CharacterService._instance = new CharacterService()
+    }
+    return CharacterService._instance
+  }
+
+  public async getEntity(id: string):Promise<Document | null> {
+    return await characters.findOne({ ownerId: id }, (err: Error, characterFound) => {        
+      if (characterFound) return characterFound
+      else throw err
+    })
+  }
+
+  public setEntity(id: string, characterToInsert: ICharacter): void {
+    characters.update({ ownerId: id }, characterToInsert, (err: Error, characterUpdated: any): Query<ICharacter> => {
+      if (err) throw err
+
+      return characterUpdated
```

Comment:
How this return is useful at all?

Reply:
But then I would have an unused declared and obligatory variable... How could I do something about it? I really don't need it there... x)

Reply:
Ah shoot! We're not even obliged to set the equivalent of 'characterUpdated' on an update function ! °°